### DISCUSSION TITLE
有人知道怎么把跳转页面的动画去掉吗?

### DISCUSSION THREAD

**Discussion by m0rml1n** [Tokinx/Wing#83] [讨论]
就是从左往右→的那个动画效果，
我记得作者的老版本是没有动画效果的，
喜欢他的极致简约，速度快

**Comment by Tokinx** [comment]
不会吧，页面切换动画一直都是有的。
目前动画有三种，开启PJAX（默认）、关闭PJAX（无动画）、添加背景图（简化动画）
暂时没有动画开关。

**Reply by m0rml1n** [reply]
我指的是你主题还没开源的时候，那时候你博客是没有动画效果的。
可能我好久没更新了吧，我还在0.5.25。没有这几个开关。找个时间在升级一下吧
